.storybook/config.js

```diff
@@ -0,0 +1,77 @@
+import {
+  configure,
+  addDecorator,
+  getStorybook,
+  setAddon,
+} from '@storybook/react';
+import {setConsoleOptions} from '@storybook/addon-console';
+import {withNotes} from '@storybook/addon-notes';
+import {withOptions} from '@storybook/addon-options';
+import {themes} from '@storybook/components';
+import createPercyAddon from '@percy-io/percy-storybook';
+
+import {
+  addPlaygroundStory,
+  generateStories,
+  hydrateExecutableExamples,
+} from './stories-from-readme';
+
+// addon-options
+addDecorator(
+  withOptions({
+    name: 'Shopify Polaris Storybook',
+    url: '/',
+    hierarchySeparator: /\//,
+    hierarchyRootSeparator: /\|/,
+    theme: {
+      ...themes.normal,
+      mainBorderRadius: 0,
+      mainBackground: '#f4f6f8',
+      mainTextColor: '#212B36',
+      highlightColor: '#202e78',
+      dimmedTextColor: '#637381',
+      // TODO more pretty brand colors?
```

Comment:
```suggestion
       mainTextSize: 16, // please
       dimmedTextColor : '#637381',
       successColor: '#108043',
       failColor: '#BF0711',
       warnColor: '#F49342',
```

Reply:
Have changed this, but I can't see where mainTextSize affects things - annoyingly it looks like the left panel values are hardcoded to a different value